### PR TITLE
⚠️ Remove all v1alpha1 codes

### LINKED ISSUE #1203: Remove all v1alpha1 code
Remove v1alpha1 types, conversion functions, and anything else referencing v1alpha1

/kind feature
/milestone v0.5.0
/priority important-soon
/help

### PR DESCRIPTION
**Pull request by tahsinrahman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1203

### CODE REVIEW

File: api/v1alpha2/conversion.go [hunk 1]
```diff
@@ -15,96 +15,3 @@ limitations under the License.
 */
 
 package v1alpha2
```
Comment (ncdc) [inline api/v1alpha2/conversion.go]: Remove this file?

Comment (tahsinrahman) [reply api/v1alpha2/conversion.go]: done! thanks!

**Pull request merged**